module-2/backend/week_two.md

```diff
@@ -5,17 +5,45 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+  A database which can utilize relationships between sets of data.
+  
 2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+  ActiveRecord Methods
+
 3. What do they allow you to do?
+  Establish relationships between sets of data, based upon the assumption that there is only one of something to many of
+  another.  These methods allow us to automatically make those connections within the ActiveRecord Database.
+
 4. What's the difference between agile workflow and waterfall method?
+   Agile workflow works on all aspects of the project at small intervals.  Whereas, waterfall does each portion of the process
+   in signifigantly larger "chunks".
+  
 5. What is the difference between `#new` and `#create`?
+  #new creates and instance of an ActiveRecord object, but is not saved to ther database.  Whereas #create does just that.  It
+  both creates a new instance Object of the data, and also saves it in one command.
+
 6. At a basic level, what does cURL allow you to do?
+   Data transfer by script...?  I think ... ?
+  
 7. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
+  One student for every teacher.  Many students for every teacher.  Thus...  Teacher has_many :students, and Students
```

Comment:
a student would `belongs_to :teacher` -> notice that `teacher` is singular here. because a student can only belong to one teacher